frontend/src/components/CodeOfConduct.jsx

```diff
@@ -0,0 +1,138 @@
+                                <p className="card1__para">Be respectful of people’s writng(s), comment(s) and or feedback. Respect people’s confidentiality. </p>
+                            </div>
+
+                        </div>
+
+                        <div className="cell large-6">
+
+                            <div className="card2">
+                                <h2 className="card2__title">02</h2>
+                                <h2 className="card2__head">Be Kind</h2>
+                                <p className="card2__para">Individuals share vulnerable thoughts and experiences. Please be kind and respectful to them. </p>
+                            </div>
+
+                        </div>
+                   
+                </div>    
+
+                <div className="grid-x grid-margin-x">
+                        
+                            <div className="cell large-6">
+
+                                <div className="card3">
+                                    <h2 className="card3__title">03</h2>
+                                    <h2 className="card3__head">No offense</h2>
+                                    <p className="card3__para">Feminist Bible is a child-friendly and family-oriented community website. Abusive, offensive and disrespectful language in comments feedback and website content should be reported to Feminist Bible team. </p>
+                                </div>
+
+                            </div>
+
+                            <div className="cell large-6">
+
+                                <div className="card4">
+                                    <h2 className="card4__title">04</h2>
+                                    <h2 className="card4__head">Report</h2>
+                                    <p className="card4__para">If you are having a problem with another account member, please send our Feminist Bible team a message containing as much information on the issues and or incident to address the situation adequately. </p>
+                                </div>
+
+                            </div>
+                        
+                </div>
+            </div>
+        </div>
+
+
+        <div className="guidelines-section">
+
+            <h2 className="guidelines-section__heading">Community Guidelines</h2>
+
+            <div className="grid-container">
+                <div className="grid-x grid-margin-x">
+
+                    <div className="cell large-4">
+                        <div class="block1">
+
+                            <img src={CodeofConduct1} alt="CodeofConduct1"  className="block1__image"/>
+
+                            <p className="block1__text">Equity is essential because it allows others to treat individual wants respectfully</p>
+
+                        </div>
+                    </div>
+
+                    <div className="cell large-4">
+                        <div class="block2">
```

Comment:
replace block1, block2, etc with `guidelines-card`

The classes can be 
`guidelines-card__image`
`guidelines-card__text`

Reply:
done